### ISSUE TITLE
The gem is spamming deprecation warnings in tests

### ISSUE THREAD

**Issue by mdesantis** [tcocca/acts_as_follower#91]
ruby 2.4.0
rails 5.0.1
acts_as_follower master@c5ac7b9
rspec 3.5.0

When I run `rspec` I get a lot of these deprecation warnings (a lot is one per test):

```
DEPRECATION WARNING: Setting custom parent classes is deprecated and will be removed in future versions. (called from parent_class_name at /home/maurizio/.asdf/installs/ruby/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/acts_as_follower-c5ac7b9601c4/lib/acts_as_follower/follower_lib.rb:10)
```

**Comment by brchristian** [comment]
@mdesantis I believe #89 should address this.

**Comment by georgestam** [comment]
Hi everyone! Not sure if this issue is solved... 

ruby 2.4.0
rails 5.0.1
acts_as_follower master@c5ac7b9
rspec 3.5.0

I get the following when runing `rspec`: 

```DEPRECATION WARNING: Setting custom parent classes is deprecated and will be removed in future versions. (called from parent_class_name at /usr/local/opt/rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/acts_as_follower-c5ac7b9601c4/lib/acts_as_follower/follower_lib.rb:10)```

**Comment by azeemh** [comment]
i've been getting this error in both rails 5.1.3 and 5.2 projects. attached is the current error it's always the same lines and same error about the deprecated custom parent class name.

ruby 2.4.1
rails  5.1.3 error code:

DEPRECATION WARNING: Setting custom parent classes is deprecated and will be removed in future versions. (called from parent_class_name at /home/azeem/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/acts_as_follower-c5ac7b9601c4/lib/acts_as_follower/follower_lib.rb:10)
DEPRECATION WARNING: Setting custom parent classes is deprecated and will be removed in future versions. (called from parent_class_name at /home/azeem/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/acts_as_follower-c5ac7b9601c4/lib/acts_as_follower/follower_lib.rb:10)

**Comment by brchristian** [comment]
@azeemh I believe #89 should address this.

**Comment by theterminalguy** [comment]
Did anyone find a way to get rid of the deprecation warnings, still having the same issues. 

```ruby
ruby '2.3.7'
rails '5.0.0'
rspec '3.5.4' 
acts_as_follower '0.2.1'
```

Between, I believe my deprecation warning is caused by  [this](https://github.com/tcocca/acts_as_follower/blob/master/lib/acts_as_follower/follower_lib.rb#L36) I think it should be `return DEFAULT_PARENTS if ActsAsFollower.custom_parent_classes.empty?`

**Comment by brchristian** [comment]
@theterminalguy You are correct and I submitted #93 containing this fix. Unfortunately we are still waiting after almost 2 years for #93 to be merged.

**Comment by ughstudios** [comment]
> @theterminalguy You are correct and I submitted #93 containing this fix. Unfortunately we are still waiting after almost 2 years for #93 to be merged.

Can someone fork this gem and merge your fix into their fork? Then we simply re-release the gem  under a new name?

**Comment by azeemh** [comment]
it's still happening. i've updated from rails 5 to rails 7 but i still have to see this same error . fyi it's 2024 and i first commented on this in 2018. it's been 6 years
```
DEPRECATION WARNING: Setting custom parent classes is deprecated and will be removed in future versions. (called from parent_class_name at /home/azeem/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/bundler/gems/acts_as_follower-c5ac7b9601c4/lib/acts_as_follower/follower_lib.rb:10)
DEPRECATION WARNING: Setting custom parent classes is deprecated and will be removed in future versions. (called from parent_class_name at /home/azeem/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/bundler/gems/acts_as_follower-c5ac7b9601c4/lib/acts_as_follower/follower_lib.rb:10)
```
hoping the fix gets merged soon. thanks
